pkgs/development/libraries/live555/default.nix

```diff
@@ -10,17 +10,20 @@
 
 stdenv.mkDerivation rec {
   pname = "live555";
-  version = "2023.05.10";
+  version = "2023.06.20";
 
   src = fetchurl {
     urls = [
       "http://www.live555.com/liveMedia/public/live.${version}.tar.gz"
       "https://download.videolan.org/contrib/live555/live.${version}.tar.gz"
       "mirror://sourceforge/slackbuildsdirectlinks/live.${version}.tar.gz"
+      "https://src.rrz.uni-hamburg.de/files/src/live555/live.${version}.tar.gz"
```

Comment:
Since this mirror contains more tarballs than the others, it is better to put it as the second.

On the other hand it is just a "fixup". No need to worry.